packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -277,6 +277,47 @@ correct log retention period (never expire, by default).
 *Further note* that, if the log group already exists and the `logRetention` is not set, the custom resource will reset
 the log retention to never expire even if it was configured with a different value.
 
+### FileSystem Access
+
+You can configure a function to mount an Amazon Elastic File System (Amazon EFS) file system to a local directory. 
+
+The following sample enables the lambda function to mount the filesystem from Amazon EFS via the `accessPoint` to 
+the local `/mnt/msg` directory.
+
+```ts
+const accessPoint = new efs.AccessPoint(stack, 'AccessPoint', {
+  fileSystem,
+  createAcl: {
+    ownerGid: '1000',
+    ownerUid: '1000',
+    permissions: '755',
+  },
+  path: '/lambda',
+  posixUser: {
+    uid: '1000',
+    gid: '1000',
+  },
+});
+
+const fn = new lambda.Function(stack, 'MyLambda', {
+  code,
+  handler,
+  runtime,
+  vpc,
+  vpcSubnets: {
+    subnetType: ec2.SubnetType.PRIVATE,
+  },
+  securityGroups: fileSystem.connections.securityGroups,
+  filesystems: {
+    filesystem: lambda.LambdaFileSystem.fromEfsFileSystem(accessPoint),
+    localMountPath: '/mnt/msg',
+  },
```

Comment:
1. Rename `lambda.LambdaFileSystem` to `lambda.FileSystem` (no need to repeat "lambda").
2. Rename `fromEfsFileSystem` to `fromAccessPoint` or `fromEfsAccessPoint`.

Can we move the localMountPath as a positional argument of the static method and avoid the extra nesting?

```ts
filesystems: [ lambda.FileSystem.fromAccessPoint(accessPoint, '/mnt/msg'[, options]) ]
```

This reads much cleaner: "file system from access point to /mnt/msg".

Reply:
OK. I think we probably have to use `fromEfsAccessPoint` in case Lambda will support more different filesystem types with different types of  access points in the future.